public/components/agents/fim/inventory/table.tsx

```diff
@@ -145,7 +145,7 @@ export class InventoryTable extends Component {
     const filters = filtersToObject(this.props.filters);
     const filter = {
       ...filters,
-      offset: pageIndex * pageSize,
+      formatUIDate: pageIndex * pageSize,
```

Comment:
The `offset` filter shouldn't change.

Reply:
This change does the integrity monitoring files tables doesn't work correctly when changing the page.
![image](https://user-images.githubusercontent.com/34042064/111614709-2e0d9c00-87e0-11eb-82e0-e57621b051fe.png)

Reply:
Replaced `formatUIDate` by `offset`, but cannot reproduce the error. Is working properly for me